src/lib/markdown_generators.js

```diff
@@ -33,22 +33,23 @@ const generateMarkdownFromJson = courseData => {
     data: courseHomeMarkdown
   })
   let menuIndex = 10
-  courseData["course_pages"].forEach(page => {
-    if (page["text"]) {
-      const pageName = page["short_url"]
-      let courseSectionMarkdown = generateCourseSectionFrontMatter(
-        page["title"],
-        menuIndex
-      )
-      courseSectionMarkdown += generateCourseSectionMarkdown(page, courseData)
-      const sectionData = {
-        name: `sections/${pageName}.md`,
-        data: courseSectionMarkdown
+  markdownData.concat(
```

Comment:
Should be `return markdownData.concat(...`

Reply:
Oh I missed the `return markdownData` below. However `concat` will not mutate the array so you need to `return markdownData.concat` here or `markDownData = markdownData.concat(...)` or `.push` the items one by one

Reply:
Another way is to use the spread operator:

    return [
      {
        name: "_index.md",
        data: courseHomeMarkdown
      },
      ...coursePagesWithText.map(...)
    ]